packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -371,6 +376,48 @@ export class AndroidEmulatorDevice extends DeviceBase {
     ]);
   }
 
+  async mirrorNativeLogsDispose() {
+    this.nativeLogsOutputChannel?.dispose();
+    this.nativeLogsProcess?.kill();
+  }
+
+  async mirrorNativeLogs(build: AndroidBuildResult) {
+    const startTime = strftime("%F %T.000", new Date());
+
+    const extractPidFromLogcat = async () =>
+      new Promise<string>((resolve, reject) => {
+        const process = exec(ADB_PATH, ["logcat"]);
+
+        lineReader(process).onLineRead((line) => {
+          const regex = new RegExp(`Start proc ([0-9]{4}):${build.packageName}`);
+
+          if (regex.test(line)) {
+            const groups = regex.exec(line);
+            const pid = groups?.[1];
+            process.kill();
+
+            if (pid) {
+              resolve(pid);
+            } else {
+              reject(new Error("PID not found"));
+            }
+          }
+        });
+      });
+
+    const nativeLogsOutputChannel = window.createOutputChannel("Radon IDE (Android Native Logs)", {
+      log: true,
+    });
+
+    const pid = await extractPidFromLogcat();
+    const process = exec(ADB_PATH, ["logcat", "--pid", pid, "-T", startTime]);
```

Comment:
what's the point of using `-T` parameter here? I understand that the time we set is from before the process is launched. Given PID will be unique I'd expect adb to only provide logs that are relevant to us this way and cropping it with -T shouldn't change anything